src/pages/Mypage/components/Common/Thumbnail.jsx

```diff
@@ -1,11 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
+import { Link } from 'react-router-dom';
 import Img from './Img';
 
 const Thumbnail = ({ src, alt, size }) => {
+  let [user, setUser] = useState([]);
+
+  user = [{ username: 'abc', id: 0 }];
```

Comment:
```suggestion
  const [user, setUser] = useState([{ username: 'abc', id: 0 }]);
```
state는 불변성을 지켜야 하기에 let 보다는 const로 선언하고
초기값을 주고 싶을 때는 위와 같은 방법으로 추가해주세요

Reply:
수정했습니다 감사합니다!